packages/next/build/index.ts

```diff
@@ -343,6 +347,41 @@ export default async function build(
     'utf8'
   )
 
+  const manifestPath = path.join(
+    distDir,
+    isLikeServerless ? SERVERLESS_DIRECTORY : SERVER_DIRECTORY,
+    PAGES_MANIFEST
+  )
+
+  const requiredServerFiles = {
+    version: 1,
+    config: {
+      ...config,
+      compress: false,
+      configFile: undefined,
+    },
+    files: [
```

Comment:
The .map can just happen on the array right, no need to spread a new array. Especially as map created a new array already